src/fortanix/Logo/Logo.tsx

```diff
@@ -2,22 +2,23 @@
 |* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
 |* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { classNames as cx, type ComponentProps } from '../../../util/componentUtil.ts';
+import { classNames as cx, type ComponentProps } from '../../util/componentUtil';
 import * as React from 'react';
 
-import logoImage from '../../../assets/fortanix/fortanix_logo_icon.svg';
+import logoImage from  '../../assets/fortanix/fortanix_logo_icon.svg';
 import cl from './Logo.module.scss';
 
 
 export type LogoProps = ComponentProps<'figure'> & {
-  subtitle: React.ReactNode,
+  subtitle?: undefined | React.ReactNode,
   subtitleTrademark?: undefined | boolean,
 };
 
 /**
  * Logo component.
  */
 export const Logo = ({ subtitle, subtitleTrademark, ...propsRest }: LogoProps) => {
+  const showSubtitle = !!subtitle
```

Comment:
Missing semicolon here.

Also, the boolean is not really necessary, you can just do `subtitle && ...` below